audit_log/api/views.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import mixins, viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from audit_log.api.serializers import AuditLogSerializer
+from audit_log.models import AuditLog
+
+
+class AuditLogViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+    queryset = AuditLog.objects.all()
+    serializer_class = AuditLogSerializer
+    permission_classes = [IsAuthenticated]
+    http_method_names = ["post"]
```

Comment:
Is this needed when using `CreateModelMixin` ?

Reply:
I ran the GET test without any `http_method_names` and the response was 405 (Method Not Allowed) so seems that is not needed.

Reply:
This is removed now.